e2e/test-06-tagged-backup.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+load "lib/utils"
+load "lib/detik"
+load "lib/k8up"
+
+# shellcheck disable=SC2034
+DETIK_CLIENT_NAME="kubectl"
+# shellcheck disable=SC2034
+DETIK_CLIENT_NAMESPACE="k8up-e2e-subject"
+# shellcheck disable=SC2034
+DEBUG_DETIK="true"
+
+@test "Given a PVC, When creating a Backup with tags, Then expect Restic snapshot with tags" {
+	expected_content="expected content: $(timestamp)"
+	expected_filename="expected_filename.txt"
```

Comment:
is it really worth having and maintaining an e2e test for this (presumably rather rare) use case?
Is an integration test or even just unit test not enough to verify that the container contains some args?

Each e2e test case adds 1-2 minutes of pipeline run

Reply:
I've switched to an integration test.